.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "cSpell.words": ["transceive"]
+    "cSpell.words": ["transceive", "sensirion"]
```

Comment:
ha I'd stopped using cspell because it kept flagging quirky words but its actually still quite useful if you can create an ignore list like this